ui/src/app/help/components/help.tsx

```diff
@@ -23,14 +23,6 @@ export const Help = () => (
                             <a href={`download/argocd-linux-amd64`} className='user-info-panel-buttons argo-button argo-button--base'>
```

Comment:
Depending on image type download URL should be `argocd-linux-amd64` or `argocd-linux-arm`. You can inject architecture into global variables during build using webpack. See example here:

https://github.com/argoproj/argo-cd/blob/09c5d5f42abe2a8a7a36946cd69c15c625ec98ae/ui/src/app/webpack.config.js#L63

Reply:
Updated but I've never done anything with node so please help double check.